compatibility/history-proxy/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12.18.2 as basis
+
+#Set the Dojot version
```

Comment:
Use multistage like in another's service in here

Reply:
https://github.com/dojot/dojot/blob/master/storage/time-series/influxdb/retriever/Dockerfile

Reply:
You will probably have to add dockerfile dependencies, since you are using sdk and it depends on those dependencies, even if you are going to use only the log.